system-x/services/cxf-soap/src/main/java/software/tnb/cxf/soap/service/CxfSoap.java

```diff
@@ -0,0 +1,35 @@
+package software.tnb.cxf.soap.service;
+
+import software.tnb.common.account.NoAccount;
+import software.tnb.common.client.NoClient;
+import software.tnb.common.deployment.WithDockerImage;
+import software.tnb.common.service.Service;
+import software.tnb.cxf.soap.validation.CxfSoapValidation;
+
+public abstract class CxfSoap extends Service<NoAccount, NoClient, CxfSoapValidation> implements WithDockerImage {
```

Comment:
I'd prefer `CxfSoap` -> `CXFSoap`, wdyt?

Reply:
So i shouldn't provide validation class at all ?

Reply:
I like the CamelCase everywhere more, it is more pleasant to my eyes.

Reply:
> So i shouldn't provide validation class at all ?

the validation class should be for interacting with the service in some way - and since there is no interaction, then the validation can be omitted